StyleGuide.md

```diff
@@ -1070,7 +1070,32 @@ CREATE DATABASE movr PRIMARY REGION "gcp-us-east1" REGIONS "gcp-us-east1", "gcp-
 ```
 
 For more information about the `remote_include` tag, see the README in the [jekyll-remote-include](https://github.com/cockroachdb/jekyll-remote-include) repo.
+  
+#### Filter tabs
+  
+On some pages in our docs, there are tabs at the top of the page that will link to different pages at different hyperlinks. For example, in the [Install CockroachDB docs](https://www.cockroachlabs.com/docs/stable/install-cockroachdb.html), there are links to the Mac, Linux, and Windows pages at the top of the page.
+  
+You can use [`filter-tabs.md`](https://github.com/cockroachdb/docs/blob/master/_includes/filter-tabs.md) to simplify the process for any `cockroachcloud` docs or docs for v21.2 and later.
```

Comment:
Based on your phrasing I assume 'the old way' will still work, but its more convenient to use the new way. Be that as it may, it probably makes sense to require the new way, so I recommend:

```suggestion
Use [`filter-tabs.md`](https://github.com/cockroachdb/docs/blob/master/_includes/filter-tabs.md) to specify these tabs for any `cockroachcloud` docs or docs for CockroachDB v21.2 and later.
```